web/src/test/java/org/cbioportal/weblegacy/ApiControllerConfig.java

```diff
@@ -61,12 +61,7 @@
 @EnableWebMvc
 @ComponentScan(basePackages = {"org.mskcc.cbio.portal.web.api", "org.mskcc.cbio.portal.persistence", "org.mskcc.cbio.portal.service"})
 public class ApiControllerConfig extends WebMvcConfigurerAdapter {
-    @Override
-    public void configureMessageConverters(List<HttpMessageConverter<?>> converters) {
-        MappingJackson2HttpMessageConverter mappingJackson2HttpMessageConverter = new MappingJackson2HttpMessageConverter();
-        mappingJackson2HttpMessageConverter.setObjectMapper(new CustomObjectMapper());
```

Comment:
Remove attempt to link to a CustomObjectMapper ... not needed? And what about the connected mixins? Perhaps this is not needed for test code?